scripts/mutect2_wdl/mutect2.wdl

```diff
@@ -380,7 +380,7 @@ task Filter {
 
     # FilterByOrientationBias must come after all of the other filtering.
     if [[ ! -z "${pre_adapter_metrics}" ]]; then
-        java -Xmx4g -jar $GATK_JAR FilterByOrientationBias -A ${sep=" -A " artifact_modes} \
+        java -Xmx4g -jar $GATK_JAR FilterByOrientationBias -AM ${sep=" -AM " artifact_modes} \
```

Comment:
Are there any other WDLs out in the wild (or production) that we have to worry about breaking with this change? Who else is using this tool?

Reply:
I could not find any others in the repo.  

There are some out in the wild.  I warned CGA, since they are using a custom WDL.  Once there is support in FireCloud, they intend to use our WDL as a subworkflow.

Reply:
Most are using our WDL in conjunction with the GATK jar, so this change will be invisible to them.